tests/L0/run_optimizers/test_fused_optimizer.py

```diff
@@ -130,7 +130,7 @@ def test_multi_params(self):
             max_abs_diff, max_rel_diff = self.get_max_diff(ref_param, tst_param)
             self.assertLessEqual(max_abs_diff, self.max_abs_diff)
             self.assertLessEqual(max_rel_diff, self.max_rel_diff)
-
+            
```

Comment:
could you remove white spaces in an empty line?

Reply:
done!